src/sentry/static/sentry/app/components/search/sources/commandSource.jsx

```diff
@@ -117,9 +121,19 @@ class CommandSource extends React.Component {
   }
 }
 
+const CommandSourceWithHooks = props => (
+  <Hook name="command-palette:action">
+    {({hooks}) => <CommandSource {...props} actions={[...hooks, ...ACTIONS]} />}
+  </Hook>
+);
+
 const CommandSourceWithFeature = props => (
   <Access isSuperuser>
-    {({hasSuperuser}) => <CommandSource {...props} isSuperuser={hasSuperuser} />}
+    {({hasSuperuser}) => (
+      <React.Fragment>
```

Comment:
Do you really need React.Fragment here?

Reply:
👍 not anymore, needed it for a split second when I was trying something else, but didn't remove. Thanks

Reply:
@EvanPurkhiser can we remove this and get this merged?

Reply:
There was some other issue I was running into with this when the hook wasn't set. I'll investigate at some point, but would like to merge yeah